pegasus/sites.v3/code.org/public/private/k5survey/index.haml

```diff
@@ -117,34 +71,42 @@
       end.compact
     end.compact.flatten
   end
-
+%link(rel='stylesheet' type='text/css' href='/css/learn_tabs.css')
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))